Pcap++/header/PcapFilter.h

```diff
@@ -243,62 +243,145 @@ namespace pcpp
 	class IPFilter : public IFilterWithDirection
 	{
 	private:
-		std::string m_Address;
+		IPAddress m_Address;
 		std::string m_IPv4Mask;
 		int m_Len;
 		void convertToIPAddressWithMask(std::string& ipAddrmodified, std::string& mask) const;
 		void convertToIPAddressWithLen(std::string& ipAddrmodified) const;
+
+		bool hasMask() const { return !m_IPv4Mask.empty(); }
```

Comment:
maybe `!m_IPv4Mask.empty() && m_Address.isIPv4()` ?

Reply:
Resolving this as the current version does not have that method anymore.